Gems/ScriptCanvas/Code/Include/ScriptCanvas/Grammar/PrimitivesDeclarations.h

```diff
@@ -177,13 +177,15 @@ namespace ScriptCanvas
 #undef REGISTER_ENUM
 
         // create the Symbol strings
+        AZ_PUSH_DISABLE_WARNING_GCC("-Wunused-variable")
 #define REGISTER_ENUM(x) #x,
         static const char* g_SymbolNames[] =
         {
 #include    "SymbolNames.h"
             "<ERROR>"
         };
 #undef REGISTER_ENUM
+        AZ_POP_DISABLE_WARNING_GCC
```

Comment:
same, `[[maybe_unused]]` instead of macros.

Reply:
constexpr worked here as well